packages/abi-coder/test/type-inference-tests/array-of-enums-abi.ts

```diff
@@ -0,0 +1,79 @@
+export const arrayOfEnumsAbi = {
```

Comment:
Instead of manually creating and versioning a Typescript version of all and each `*-abi.json`, we should have a script that does this on the fly on a `pretest` step or something similar. This will reduce the noise and also ensure that nothing is outdated concerning each respective forc project.

Reply:
I agree, I was planning on adding that and removing these files when I implement the json->ts typegen aspect of the whole feature.